src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -76,20 +75,26 @@ func (repo *MarketplaceCmdRepo) installServices(
 }
 
 func (repo *MarketplaceCmdRepo) parseSystemDataFields(
-	installTempDir valueObject.UnixFilePath,
 	installDir valueObject.UnixFilePath,
 	installUrlPath valueObject.UrlPath,
 	installHostname valueObject.Fqdn,
 	installUuid string,
-	installRandomPassword valueObject.Password,
+	installTempDir *valueObject.UnixFilePath,
```

Comment:
Why tempDir became and randomPassword became a pointer? If they aren't used, whatever, don't use it, but making them a pointer won't accomplish anything just generate doubts.

Reply:
Adjusted!